Makefile

```diff
@@ -68,9 +68,9 @@ auth_gcloud:
 	gcloud auth login --update-adc
 
 jetstream_config:
-	curl -LJ -o app/experimenter/outcomes/jetstream-config.zip $(JETSTREAM_CONFIG_URL)
-	unzip -o -d app/experimenter/outcomes app/experimenter/outcomes/jetstream-config.zip
-	rm -Rf app/experimenter/outcomes/jetstream-config-main/.scripts/
+	curl -LJ -o app/experimenter/outcomes/metric-hub.zip $(JETSTREAM_CONFIG_URL)
+	unzip -o -d app/experimenter/outcomes app/experimenter/outcomes/metric-hub.zip
+	rm -Rf app/experimenter/outcomes/metric-hub-main/.scripts/
```

Comment:
I assume this part runs as part of the `check` CI step? It's not quite obvious to me why the check can't find the outcomes...

Reply:
Great, thank you!